nifi-nar-bundles/nifi-extension-utils/nifi-reporting-utils/src/main/java/org/apache/nifi/reporting/util/provenance/ProvenanceEventConsumer.java

```diff
@@ -89,16 +94,26 @@ public void setComponentTypeRegex(final String componentTypeRegex) {
         }
     }
 
-    public void addTargetEventType(final ProvenanceEventType... types) {
-        for (ProvenanceEventType type : types) {
-            eventTypes.add(type);
+    public void setComponentTypeRegexExclude(final String componentTypeRegex) {
+        if (!StringUtils.isBlank(componentTypeRegex)) {
+            this.componentTypeRegexExclude = Pattern.compile(componentTypeRegex);
         }
     }
 
+    public void addTargetEventType(final ProvenanceEventType... types) {
+        eventTypes.addAll(Arrays.asList(types));
```

Comment:
Trivial, but is there any reason to not use `Collections.addAll` here, as addTargetEventTypeExclude does?

Reply:
Nope, I think that was the original code, it only shows up as a diff here because I added a method before it. Will change it to use Collections for consistency.